### PR TITLE
Database setup

### PR DESCRIPTION
**Pull request by jcjurado3**

### CODE REVIEW

File: Gemfile
Comment (garrettgregor) [inline Gemfile:1]: all the gems! so excited to learn faker and factory_bot!

**Review by garrettgregor — APPROVED** [review]
This is fantastic progress! We've done some really great work here and are ready for deployment!

File: lib/tasks/csv_load:all.rake [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+namespace :csv_load do
+  desc "Run all CSV Imports"
+  task :all => :environment do
+    
+    ActiveRecord::Base.connection.tables.each do |t|
+      ActiveRecord::Base.connection.reset_pk_sequence!(t)
+    end
+
+    Rake::Task["csv_load:customers"].invoke
+    Rake::Task["csv_load:merchants"].invoke
```
Comment (JCBeldo) [inline lib/tasks/csv_load:all.rake:12]: So glad we got this to work and make our code DRYer.

File: app/models/invoice.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Invoice < ApplicationRecord
+  self.primary_key = :id
+  has_many :invoice_items
+  has_many :items, through: :invoice_items
+  belongs_to :customer
+  has_many :transactions
+  enum :status, ["cancelled", "in progress", "completed"]
```
Comment (JCBeldo) [inline app/models/invoice.rb:7]: Glad we updated this to all be strings so it deploys.

**Review by JCBeldo — APPROVED** [review]
Nice setup, y'all!

**Pull request merged**